autocertkit/tests/xenapi_mock.py

```diff
@@ -489,8 +499,25 @@ def __init__(self, host):
     def __getattr__(self, name):
         return self.__defaultOutput
 
-    def __defaultOutput(self, *args):
+    def __defaultOutput(self, args):
```

Comment:
This change was done to match ack plugin calls. args is actually a dictionary and is passed in plugin as it is. We always pass args, either as {} or {key1:val1 , key2:val2 ....}